bin/runenv.sh

```diff
@@ -0,0 +1 @@
+docker-compose up $1 $2 $3
```

Comment:
why wrapping docker-compose inside a .sh script? Also, is `runenv.sh` a descriptive name for what it actually does?

Reply:
Please use kebab-case for file names, as the other project files.